src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -1027,8 +1025,6 @@ const static std::unordered_map<std::string, GameCommand> ActionNameToType = {
     { "staffsetname", GameCommand::SetStaffName },
     { "staffsetorders", GameCommand::SetStaffOrders },
     { "staffsetpatrolarea", GameCommand::SetStaffPatrol },
-    { "surfacesetstyle", GameCommand::ChangeSurfaceStyle },
-    { "tilemodify", GameCommand::ModifyTile },
     { "trackdesign", GameCommand::PlaceTrackDesign },
```

Comment:
I think for these actions that have been removed from the plugin API, like `footpathplacefromtrack`, `loadorquit` and others, their `AcceptParameters` function should rather be no-ops and actually have a comment there stating why they're intentionally empty, so that people don't think we just forgot to add them a few moments from now.